CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Additions and Improvements
 - Added `benchmark` subcommand to `evmtool` [#5754](https://github.com/hyperledger/besu/issues/5754)
+- JSON output is now compact by default. This can be overridden by the new `--pretty-json-enabled` CLI option. [#5766](https://github.com/hyperledger/besu/pull/5766)
```

Comment:
```suggestion
- JSON output is now compact by default. This can be overridden by the new `--json-pretty-print-enabled` CLI option. [#5766](https://github.com/hyperledger/besu/pull/5766)
```